pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -87,7 +87,7 @@ var (
 
 // Cluster represents the Rook and environment configuration settings needed to set up Ceph mons.
 type Cluster struct {
-	clusterInfo         *cephconfig.ClusterInfo
+	ClusterInfo         *cephconfig.ClusterInfo
```

Comment:
I understand why the `config.clusterInfo` scope was changed to `config.ClusterInfo`, but why is it necessary to change the scope here?

Reply:
That's the description I wrote in the commit, hopefully, it's clear :-)

>  By exporting cluster to Cluster we can now reach it from the controller
    health check so that it can be populated with 'cluster' data, see the
    next commit for better understanding